src/banking/primitive/core/ServerSolution.java

```diff
@@ -10,13 +10,13 @@
 
 class ServerSolution implements AccountServer {
 
-	static String fileName = "accounts.ser";
-
+	static final String fileName = "accounts.ser";

```

Comment:
Oh wow, Whoop! Thanks! I shall embrace the light and pull against the presiding darkness with all my might!